clients/js/test/_setup.ts

```diff
@@ -83,12 +91,59 @@ export const signAndSendTransaction = async (
     await signTransactionMessageWithSigners(transactionMessage);
   const signature = getSignatureFromTransaction(signedTransaction);
   assertIsSendableTransaction(signedTransaction);
+  assertIsTransactionWithBlockhashLifetime(signedTransaction);
   await sendAndConfirmTransactionFactory(client)(signedTransaction, {
     commitment,
   });
   return signature;
 };
 
+export const createDefaultTransactionPlanner = (
+  client: Client,
+  feePayer: TransactionSigner
+): TransactionPlanner => {
+  return createTransactionPlanner({
+    createTransactionMessage: async () => {
+      const { value: latestBlockhash } = await client.rpc
+        .getLatestBlockhash()
+        .send();
+
+      return pipe(
+        createTransactionMessage({ version: 0 }),
+        (tx) => setTransactionMessageFeePayerSigner(feePayer, tx),
+        (tx) => setTransactionMessageLifetimeUsingBlockhash(latestBlockhash, tx)
+      );
+    },
+  });
+};
+
+export const createDefaultTransactionPlanExecutor = (
+  client: Client,
+  commitment: Commitment = 'confirmed'
+): TransactionPlanExecutor => {
+  return createTransactionPlanExecutor({
+    executeTransactionMessage: async (transactionMessage) => {
+      const signedTransaction =
+        await signTransactionMessageWithSigners(transactionMessage);
+      assertIsSendableTransaction(signedTransaction);
+      assertIsTransactionWithBlockhashLifetime(signedTransaction);
+      await sendAndConfirmTransactionFactory(client)(signedTransaction, {
+        commitment,
+      });
+      return { transaction: signedTransaction };
+    },
+  });
+};
+
+export const getSignatureFromTransactionPlanResult = (
+  result: TransactionPlanResult
+): Signature => {
+  if (result.kind === 'single' && result.status.kind === 'successful') {
+    return getSignatureFromTransaction(result.status.transaction);
+  }
+  throw new Error('Unable to get signature from transaction plan result');
+};
```

Comment:
This is a nice helper but just wanted to flag it doesn't seem you are using that in the tests at the moment.

Reply:
Oops I thought I removed that - I think it's probably not useful for the tests atm. I did it because the `signAndSendTransaction` returns a `Signature`, but we can add a specific helper later if we need one. Or just use whatever we add to Kit for this by then.